src/backend/src/main/java/com/linkedpipes/lpa/backend/helpers/StreamHelper.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedpipes.lpa.backend.helpers;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class StreamHelper {
+
+    public static String getStringFromStream(InputStream stream) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(
```

Comment:
It may be a small one, but why is there a line break? In rest of the code the lines are much longer and I think it would be better to remain consistent.